GameTests.java

```diff
@@ -49,43 +53,48 @@ public void gameKnowsWhoseTurnItIs() {
     public void playerCannotGoInUnavaliablePositions() {
         InputStream stream = new ByteArrayInputStream("1".getBytes());
         Board board = new Board("o--------");
-        Game game = new Game(stream, board);
+        ConsoleGame consoleGame = new ConsoleGame();
+        Game game = new Game(stream, board, consoleGame);
         game.assignPlayers();
         assertThat(game.playerMakesMove().getState(), is("o--------"));
     }
 
     @Test
-    public void gameEndsAfterWin() {
+    public void boardStateDoesntChangeAfterWin() {
```

Comment:
not a very useful test.  More useful would be, no further moves accepted after a win.